# Service to get joint angles of the robot
---
float64[6] joints
